apdx-validation.tex

```diff
@@ -1076,3 +1076,26 @@ \subsubsection*{Rules for the \class{Attachment} class}
 
 \printValid{The \sbol{hash} property of an \sbol{Attachment} is OPTIONAL and MAY contain a \sbol{String}.\\ Reference: \sec{sec:Attachment}}
 }
+
+\twothreezero{
+\subsubsection*{Rules for the \class{ExperimentalData} class} 
+\setcounter{sbolCtr}{13301}
+
+\printValid{An \sbol{ExperimentalData} MUST NOT have properties other than the following: \sbol{identity}, \\
```

Comment:
I'm wondering if we should include `attachments` here are well. Technically, I seems like it should be included in ALL the validation rules that deal with allowed properties and inherit from TopLevel, but in lieu of making all those changes, we could just do it here since it seems relevant.